apps/opt/components/trips/TripsList.tsx

```
@@ -0,0 +1,11 @@
+import { Trip } from '@opt/core/interfaces';
+
+import { TripCard } from './TripCard';
+
+export const TripsList = ({ trips }: { trips: Trip[] }) => {
+  return (
+    <ul className="trip-list">
+      {trips && trips.map((trip) => <TripCard key={trip.id} {...{ trip }} />)}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)